ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/DeploymentComponentVulnerabilitiesTable.tsx

```diff
@@ -95,13 +102,22 @@ function DeploymentComponentVulnerabilitiesTable({
                     index !== componentVulns.length - 1
                         ? { borderBottom: '1px solid var(--pf-c-table--BorderColor)' }
                         : {};
+                const hasPendingException = componentVulns.some(
+                    (vuln) => vuln.pendingExceptionCount > 0
+                );
 
                 return (
                     <Tbody key={`${image.id}:${name}:${version}:${vulnerabilityId}`} style={style}>
                         <Tr>
                             <Td>
                                 {image.name ? (
-                                    <ImageNameTd name={image.name} id={image.id} />
+                                    <PendingExceptionLabelLayout
+                                        hasPendingException={hasPendingException}
+                                        cve={cve}
+                                        vulnerabilityState={vulnerabilityState}
+                                    >
+                                        <ImageNameTd name={image.name} id={image.id} />
```

Comment:
I had done something similar in Exception Management where I appended the `Td` to the component name. I had to reuse the contents of the component in another area where the `Td` didn't make sense anymore (because it wasn't in a table), and decided to remove it. What do you think about doing the same for this? Since there is a `<Td>` component two levels above, maybe it makes sense to just call this `ImageName` or something equivalent. Not entirely necessary though and even if we leave it like this, I think it makes sense. Just a little thought I had as I looked over the code

Reply:
Yeah this makes sense. I think at one point we had these components that actually contained the `<Td>` inside them, but now that they are stand alone the `Td` in the name isn't relevant.

I could batch rename all of these in a follow up.